### PR TITLE
Map Renderer Component

### LINKED ISSUE #44: Create map component

### PR DESCRIPTION
**Pull request by MtheDV**
Create the map component to be used in the map page

Closes #44

### CODE REVIEW

File: src/components/App/MapRenderer/MapRenderer.module.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.mapRenderer {
+  width: 50rem;
+  height: 50rem;
+}
```
Comment (Opeyem1a) [inline src/components/App/MapRenderer/MapRenderer.module.scss:4]: EOF newline

File: src/components/App/MapRenderer/MapRenderer.module.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.mapRenderer {
+  width: 50rem;
```
Comment (Opeyem1a) [inline src/components/App/MapRenderer/MapRenderer.module.scss:2]: How does this behave on smaller screen sizes? (Even if the answer is "badly", I think dealing with that should be on a separate branch so we can get this branch merged faster)

File: src/components/App/MapRenderer/index.tsx [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+const MapRenderer = ({ markers }: MapRendererProps) => {
+  const [geocoderSearchValue, setGeocoderSearchValue] = useState<string>('');
+  const [geocoderGeocodedPlace, setGeocoderGeocodedPlace] = useState<string>('');
+  let geocoder = new Geocoder({
+    key: MAPTILER_ACCESS_TOKEN
+  });
+  
+  const [center, setCenter] = useState<Anchor>([
+    49.882114, -119.477829
```
Comment (Opeyem1a) [inline src/components/App/MapRenderer/index.tsx:33]: I think this is the branch is the one you mentioned still has things to be worked on, so I'll leave the review here for now

**Review by Opeyem1a — CHANGES_REQUESTED** [review]

File: src/components/App/MapRenderer/index.tsx [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+    setZoom(marker.zoom);
+  };
+  
+  const onGeocoderSearch = (): void => {
+    setGeocoderSearchValue(geocoderSearchValue);
+    geocoder.geocode(geocoderSearchValue).then((results) => {
+      const response = results.features[0];
+      setGeocoderGeocodedPlace(`${response.place_name} ${response.center}`);
+      setCenter([response.center[1], response.center[0]]);
+      setZoom(13);
```
Comment (ketphan02) [inline src/components/App/MapRenderer/index.tsx:59]: The defaultZoom can also be added here too

File: src/components/App/MapRenderer/index.tsx [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+            (anchor: [{yourAnchor.anchor[0]}, {yourAnchor.anchor[1]}], zoom: {yourAnchor.zoom}),
+          </button>
+        )}
+        ]
+      </pre>
+    </div>
+  );
+};
+
+export default MapRenderer;
```
Comment (ketphan02) [inline src/components/App/MapRenderer/index.tsx:125]: Add a line here too

**Review by ketphan02 — CHANGES_REQUESTED** [review]

File: src/components/App/MapRenderer/MapRenderer.module.scss (hunk 2 above)
Comment (MtheDV) [reply src/components/App/MapRenderer/MapRenderer.module.scss:2]: it is

File: src/components/App/MapRenderer/index.tsx (hunk 3 above)
Comment (MtheDV) [reply src/components/App/MapRenderer/index.tsx:33]: this is being updated by another branch, I guess we can stale this one

**Pull request closed without merging**